config/env.config.js

```diff
@@ -55,5 +57,77 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
+    this.apply = function(compiler) {
+        compiler.plugin('compilation', function(compilation) {
+            compilation.plugin('html-webpack-plugin-before-html-processing', function(htmlPluginData, callback) {
+                let headOpeningTag = '<head>'; 
+                let htmlHead = htmlPluginData.html.match(headOpeningTag);
+                if (htmlHead) {
+                    htmlHead = htmlHead.index;
+                    htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + headOpeningTag +
+                    `
+                    <script>
+                    (function() {
+                        function getParameterByName(name) {
+                            var url = window.location.search;
+                            var queryExp = new RegExp("[\\?&]"+name+"=([^&#]*)", "i");
+                            var match = queryExp.exec(url);
+                            if (match && match.length > 1) {
+                                return match[1];
+                            }
+                            return null;
```

Comment:
This should not be an issue, as it is checked in the function a few lines down, via the statement

`
(getParameterByName("originEnvironment") || "").toLowerCase();
`

Wanted the internal function to return null to explicitly signal that the parameter did not exist.